modules/apps/foundation/portal-search/portal-search-web/src/main/resources/META-INF/resources/facets/view/users.jsp

```diff
@@ -19,9 +19,13 @@
 <c:if test="<%= !termCollectors.isEmpty() %>">
```

Comment:
Let's use `displayContext.isRenderNothing()` and disregard `termCollectors` entirely. Also, instead of `c:if` let's use `c:choose` and a hidden field to preserve the param across renders even if an empty result occurs inbetween. See: https://github.com/liferay/liferay-portal/blob/45e0d00a67a6cdfeb76f2819cdc0f810f8fd697f/modules/apps/foundation/portal-search/portal-search-web/src/main/resources/META-INF/resources/facets/view/scopes.jsp#L27-L31